api/src/main/java/io/minio/errors/BucketPolicyTooLargeException.java

```diff
@@ -21,8 +21,9 @@
  */
 @SuppressWarnings({"WeakerAccess", "unused"})
 public class BucketPolicyTooLargeException extends MinioException {
-  private final String bucketName;
+  private static final long serialVersionUID = 6435208839560441168L;
```

Comment:
could you tell me about his declaration?

Reply:
Exceptions are all Serializable, so they should include an explicit version. Actual findbugs error is  "SnVI".

References:
http://findbugs.sourceforge.net/bugDescriptions.html#SE_NO_SERIALVERSIONID
http://lmgtfy.com/?q=serialVersionUID
http://lmgtfy.com/?q=serialVersionUID+findbugs

Reply:
With the reference https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it, I think we should ignore the warning.  Similarly it looks odd to define only to this exception.

Reply:
Ok, I definitely agree that they should be everywhere or nowhere. 

It's not a extremely important to use manually set serialVersionUID as it's more likely that they won't be updated when a serializable class changes than that they cause unnecessary. For some reason I don't get findbugs errors on this one anymore, and I never got on the other classes.